Download/index.mixer

```diff
@@ -176,8 +176,8 @@ total    201147 ms (27195 ms GC) and 5.78GB allocated
 in this order:
 </p>
 <ul>
-<li>Look at the "If Things Go Wrong" section of 
```

Comment:
Oops. That was surely wrong, link saying GAP 4.10.2 pointing to 4.9.1.  Pointing it to master, OTOH, may point people to INSTALL referring to things not yet in their version.  Cf. https://www.gap-system.org/Releases/index.html which correctly refers to https://github.com/gap-system/gap/blob/v4.10.2/INSTALL.md. I am not sure how to properly organise this on _this_ page though to avoid misleading users. Maybe remove the hyperlink and say `INSTALL.md` of your GAP installation? That will, in its turn, confuse novices who don't know how to open `.md` files on their computers....

Reply:
Once you merge PR #146 then I'll update this PR to link here to the right version.

Reply:
#146  merged, please go ahead

Reply:
Done

Reply:
@fingolfin thanks